### DISCUSSION TITLE
Allow MSI installer to run only by Administrator

### DISCUSSION THREAD

**Discussion by saadsaeed01** [wixtoolset/issues#8083] [Questions]
Hello

Is there a property in WiX toolset which forces the MSI installer to run only by Administrator?

I want to restrict normal accounts from running the MSI installer.

**Comment by chrpai** [comment]
https://learn.microsoft.com/en-us/windows/win32/msi/privileged

**Reply by saadsaeed01** [reply]
Hello @chrpai 

I have tried using this property but it doesn't not give any errors or warning. I am trying to run the installer from the normal account

![image](https://github.com/wixtoolset/issues/assets/33133831/d626a737-64cc-4b76-afa0-2ede90ab539c)

Am I using this property correctly?

**Comment by chrpai** [comment]
MSI already defines the property.  You need to use the property in a condition.  Such as  `Privileged or Installed`.
https://wixtoolset.org/docs/v3/xsd/wix/condition/

**Comment by saadsaeed01** [comment]
Hi @chrpai 

Thanks for the answer!

I solved this issue through **custom action** and it is working fine
